iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractNodeBreakerTest.java

```diff
@@ -165,40 +312,110 @@ private static Network createIsolatedLoadNetwork() {
     @Test
     public void connectDisconnectRemove() {
         Network network = createNetwork();
-        VoltageLevel.NodeBreakerView topo = network.getVoltageLevel("VL").getNodeBreakerView();
-        Load l = network.getLoad("L");
-        Generator g = network.getGenerator("G");
-
-        // generator is connected, load is disconnected
-        assertTrue(topo.getOptionalTerminal(2).isPresent());
-        assertTrue(topo.getOptionalTerminal(3).isPresent());
-        assertNotNull(g.getTerminal().getBusView().getBus());
-        assertNull(l.getTerminal().getBusView().getBus());
-        assertTrue(g.getTerminal().isConnected());
-        assertFalse(l.getTerminal().isConnected());
-
-        // connect the load
-        assertTrue(l.getTerminal().connect());
-
-        // check load is connected
-        assertTrue(topo.getOptionalTerminal(2).isPresent());
-        assertNotNull(l.getTerminal().getBusView().getBus());
-        assertTrue(l.getTerminal().isConnected());
 
-        // disconnect the generator
-        g.getTerminal().disconnect();
+        VoltageLevel.NodeBreakerView topo = network.getVoltageLevel("VL1").getNodeBreakerView();
+        Load l1 = network.getLoad("L1");
+        Generator g1 = network.getGenerator("G1");
+        Generator g2 = network.getGenerator("G2");
+
+        // generator 2 is connected, load and generator 1 are disconnected
```

Comment:
Isn't the load connected, here?

Reply:
yep, load and g2 are connected, g1 is disconnected, I changed that.